scripts/linters/js_ts_linter.py

```diff
@@ -901,6 +901,91 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_comments(self):
+        """This function ensures that comments follow correct style. Below are
+        some formats of correct comment style:
+        1. A comment can end with the following symbols: ('.', '?', ';', ',',
+        '{', '^', ')', '}', '>'). Example: // Is this is comment?
+        2. If a line contain any of the following words or phrases('@ts-ignore',
+        '--params', 'eslint-disable', 'eslint-enable', 'http://', 'https://')
+        in the comment.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT(
+                'Starting comment checks\n'
+                '----------------------------------------')
+        summary_messages = []
+        files_to_check = self.all_filepaths
+        allowed_terminating_punctuations = [
+            '.', '?', ';', ',', '{', '^', ')', '}', '>']
+
+        # We need to ignore phrases below because if we use punctuation with
```

Comment:
I think you need to explain more clearly what "ignored" means in this context. Are we stipulating that these cannot occur at the end of a comment? Or that comments that contain any of these phrases are ignored? Or that individual lines which contain these phrases are ignored? Or something else? Please be explicit about what this means. You're not actually ignoring the phrases -- what "ignore phrase" means is that you treat the original comment as though that particular phrase was deleted, and process the rest of the comment accordingly.

Also, isn't this too broad a check, especially if all you want to do is to not check for a period at the end? I think you should make this check more specific so that this becomes something like allowed_comment_endings_without_period, rather than a broad exclusion. Or, if some phrases need a broad exclusion, then have two or more separate lists and apply the relevant specific check for each list.

Reply:
Done.